File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let arr = x.toString().split('');
    console.log(arr)
    let i = 0;
    let j = arr.length - 1;
    
    while (i < arr.length) {
        if (arr.at(i) !== arr.at(j)) {
            return false;
        } else {
            i++;
            j--;
        }
    }
    return true;
    
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        s = str(x)
        i, j = 0, len(s) - 1
        while i < j:
            if s[i] != s[j]:
                return False
            i += 1
            j -= 1
        return True
        